### DISCUSSION TITLE
Contact assignment rather to contact group than to contact

### DISCUSSION THREAD

**Discussion by empusas** [netbox-community/netbox#9088] [Ideas]
While working on a plugin I wanted to use the contact assignment. Thinking about it I recognized that usually not a single contact is assigned to task, rather it is a group.

Let's take the device model as example.

For example if Joe is assigned as contact for switches in Dallas, then usually Joe is part of a team or he can never get sick or go on vacation.
If Jim is in the same team, has the same task to look after those switches in Dallas, you can simply assign Jim as well as contact to every switch in Dallas.
But thinking of a big team(10, 20 or 100) and many devices that can be a tedious task and also would completely mess up the view of each device to have that many contacts.

So assigning a contact group to a device, tenant etc. would be beneficial.

**Comment by candlerb** [comment]
Raised before as #8359
